sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -734,6 +1002,9 @@ export class TextAnalysisClient {
     analyze<ActionName extends "LanguageDetection">(actionName: ActionName, documents: string[], countryHint?: string, options?: AnalyzeActionParameters<ActionName> & TextAnalysisOperationOptions): Promise<AnalyzeResult<ActionName>>;
     analyze<ActionName extends AnalyzeActionName = AnalyzeActionName>(actionName: ActionName, documents: TextDocumentInput[], options?: AnalyzeActionParameters<ActionName> & TextAnalysisOperationOptions): Promise<AnalyzeResult<ActionName>>;
     analyze<ActionName extends AnalyzeActionName = AnalyzeActionName>(actionName: ActionName, documents: string[], language?: string, options?: AnalyzeActionParameters<ActionName> & TextAnalysisOperationOptions): Promise<AnalyzeResult<ActionName>>;
+    beginAnalyzeBatch(actions: AnalyzeBatchAction[], documents: string[], language?: string, options?: BeginAnalyzeBatchOptions): Promise<AnalyzeBatchPoller>;
+    beginAnalyzeBatch(actions: AnalyzeBatchAction[], documents: TextDocumentInput[], options?: BeginAnalyzeBatchOptions): Promise<AnalyzeBatchPoller>;
+    createAnalyzeBatchPoller(serializedState: string, options?: CreateAnalyzeBatchPollerOptions): Promise<AnalyzeBatchPoller>;
```

Comment:
is this identical to calling `beginAnalyzeBatch` with `serializedState` as the `resumeFrom` value? I'm fine with the extra convenience, just wanted to confirm my understanding of the implementation.

Reply:
Yes. However, `createAnalyzeBatchPoller` does not expect any other inputs whereas `beginAnalyzeBatch` still expects the lists of actions and documents as input regardless. That made me think that the `resumeFrom` parameter is not needed now that we have a standalone convenient method.

I wonder what are your thoughts about standardizing this pattern? On one hand, creating a poller is no longer awkward, but on the other, it is an extra method per LRO which could be make intellisense experience for searching for methods a bit challenging, especially if creating pollers is considered an advanced scenario.

Reply:
I quite like this pattern since it addresses the badness of `resumeFrom`. 

I think @johanste was complaining about `resumeFrom` being a pain because of the required argument issue. Or maybe it was @JonathanGiles ? Either way, perhaps a good thing to poke the architect alias about.

Reply:
I like this a lot because resumeFrom is a huge pain to work with in FR as well. You have to provide a bunch of information to the methods that is completely discarded if you use a resume token.

Reply:
Maybe `resumeAnalyzeBatch(resumeFrom: string, options?: CreateAnalyzeBatchPollerOptions)` is a more natural name? It parallels `beginAnalyzeBatch` nicely.

Reply:
I thought about that name, and I agree it coincides with the `begin` method but I wonder if the behavior here is accurately described by `resuming`. In my mind, `resuming` gives the impression that the thing was suspended but I don't think that is the case here. This method just creates a poller and returns it and doesn't alter the state of the LRO in any way. I am open to be convinced otherwise though.

Reply:
Yeah this was the main issue with terminology, since the verb feels like it's talking about the _operation_ when it is actually talking about the _poller_.

Maybe we could lean into that with `pollOperationBatch()` or `observeOperationBatch()` ?

Reply:
I will reach out offline to brainstorm this further.

Reply:
We landed on `restoreAnalyzeBatchPoller` for now but feel free to keep the offline discussion going.